VAMobile/src/utils/personalInformation.tsx

```diff
@@ -97,14 +93,3 @@ export const getAddressDataFromSuggestedAddress = (suggestedAddress: SuggestedAd
     addressMetaData: suggestedAddress?.meta?.address,
   }
 }
-
-export const getPhoneDataForPhoneType = (phoneType: PhoneType, profile: UserDataProfile): PhoneData | undefined => {
-  switch (phoneType) {
-    case PhoneTypeConstants.HOME:
-      return profile.homePhoneNumber
-    case PhoneTypeConstants.MOBILE:
-      return profile.mobilePhoneNumber
-    case PhoneTypeConstants.WORK:
-      return profile.workPhoneNumber
-  }
-}
```

Comment:
This function is just checking if the user has a `phoneType` saved, and returns it. This was only being used in the `deleteEmail` Redux action; right before making the API call it would call it to verify that the user had the phone type save in their profile. `EditPhoneNumberScreen` gets passed the `phoneData` prop which comes from the `contactInformation`. So just checking if `phoneData` isn't null should mean that the user has it saved on their profile. Not sure that this function call is needed to verify that